server/supernova/audio_backend/portaudio_backend.hpp

```diff
@@ -124,12 +104,40 @@ class portaudio_backend : public detail::audio_backend_base<sample_type, float,
     bool open_stream(std::string const& input_device, unsigned int inchans, std::string const& output_device,
                      unsigned int outchans, unsigned int samplerate, unsigned int pa_blocksize, int h_blocksize) {
         int input_device_index, output_device_index;
-        if (!match_device(input_device, input_device_index) || !match_device(output_device, output_device_index))
+
+        input_device_index = GetPaDeviceFromName(input_device.c_str(), true);
+        output_device_index = GetPaDeviceFromName(output_device.c_str(), false);
+
+        // check device selection, select defaults if needed
+        std::cout << std::endl;
+        PaError checked =
+            TryGetDefaultPaDevices(&input_device_index, &output_device_index, inchans, outchans, samplerate);
+        std::cout << std::endl;
+
+        std::cout << "opening audio devices:" << std::endl;
```

Comment:
i noticed that supernova output is inconsistent with capitalizing first letters of messages like these, but the majority seem to be capitalized, can you make sure to do that with all the messages added/modified in this PR?